### DISCUSSION TITLE
(Piano song)Home from undertale

### DISCUSSION THREAD

**Discussion by ForceDog** [Monika-After-Story/MonikaModDev#7800] [general - piano notes]
So I tried to make a piano song json with the piano notes that @Abc2557 posted on issues.I don't know if it's right since it's my first time trying to do this.
[pianosong.json.txt](https://github.com/Monika-After-Story/MonikaModDev/files/4571757/pianosong.json.txt)

**Comment by ForceDog** [comment]
{
    "name": "Home",
    "verse_list": [0, 10, 20, 30,],
    "pnm_list": [
        {
            "text": "(playing home from undertale)",
            "style": "monika_credits_text",
            "notes": [
                "F4",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "D5",
                "G4",
                "A4SH",
                "C5",
                "F5",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "F5",
                "F4",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "D5",
                "G4",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "D5",
                "A4",
                "A4SH",
                "C5",
                "F5",
                "A4SH",
                "C5",
                "D5"
            ],
            "express": "1eub",
            "postexpress": "1eua",
            "vis_timeout": 2.0,
            "verse": 0,
            "posttext": true
        },
        {
            "text": "(Playing home from undertale)",
            "style": "monika_credits_text",
            "notes": [
                "A4SH",
                "C5",
                "A4SH",
                "F4",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "D5",
                "G4",
                "A4SH",
                "C5",
                "D5",
                "A4SH",
                "C5",
                "D5",
                "F4SH",
                "A4SH",
                "C5",
                "D5"
            ],
            "postnotes": [
                "F4SH",
                "A4SH",
                "D5",
                "A4SH",
                "A4SH",
                "C5",
                "A4SH"
            ],
            "express": "1eub",
            "postexpress": "1eua",
            "ev_timeout": 1.0
        }
    ]
}

**Comment by ThePotatoGuy** [comment]
linking #5706
